web/components/editor.tsx

```diff
@@ -75,19 +81,34 @@ export function useTextEditor(props: {
   defaultValue?: Content
   disabled?: boolean
   simple?: boolean
+  key?: string // unique key for autosave. If set, plz call `clearContent(true)` on submit to clear autosave
```

Comment:
Hmm, having to call `editor.commands.clearContent(true)` is not that great. But I'm not sure of a better solution.

Slightly better would be a top level `reset()` returned from the hook?
Or maybe the caller should change the key after submitting?
Otherwise, this is not blocking.